### PR TITLE
Extend question headings

### PR DESCRIPTION
**Pull request by tombye**
Extends the existing `question-headings.scss` file to include code for the question and answer formatting developed as part of the Supplier Submission Portal. (now called `questions.scss`.)

See https://github.com/alphagov/digitalmarketplace-frontend-toolkit/pull/15 for an example.

Also includes some code changes missed from previous pull requests.

### CODE REVIEW

File: scss/forms/_questions.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+@import "_typography.scss";
+
+.question {
+  margin: 15px 0 0 0;
+  clear: both;
+}
+
+.first-question {
+  margin-top: 0;
+}
```
Comment (quis) [inline scss/forms/_questions.scss]: Should we use `@extend` here? I.e. in the markup, which is preferable:
1. `class="question first-question"`
2. `class="first-question"`

If 2., then I think for consistency `question-first` is more in keeping with `banner-warning`, `banner-success` etc.

File: scss/forms/_summary.scss [hunk 2]
```diff
@@ -19,6 +19,10 @@
   @include box-sizing(border-box);
 }
 
+.first-summary-item-heading {
```
Comment (quis) [inline scss/forms/_summary.scss]: Same query as here: https://github.com/alphagov/digitalmarketplace-frontend-toolkit/pull/14/files#r22858140

File: scss/forms/_questions.scss (hunk 1 above)
Comment (tombye) [inline scss/forms/_questions.scss]: I've amended the naming for consistency. I think the use of `@extend` should be implemented here but the same should be done to other parts of the CSS so I'm going to do that in a separate pull request.
